docs/website/content/v0.3/en/components/apid.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'apid'
+---
+
+When interacting with Talos, the gRPC api endpoint you will interact with directly is apid.
+Apid acts as the gateway for all component interactions.
+Apid provides a mechanism to route requests to the appropriate destination.
+
+We'll use some examples below to illustrate what apid is doing.
+
+When a user wants to interact with a Talos component via `osctl`, there are two flags that control the interaction with apid.
+The `-e | --endpoints` flag is used to denote which Talos node ( via apid ) should handle the connection.
+Typically this is a public facing server.
+*Note* Typically the endpoints are configured in the Talos config file
+The `-n | --nodes` flag is used to denote which Talos node(s) should respond to the request.
+
+For example, if a user wants to interact with `osd`, a command like `osctl -e cluster.talos.dev memory` may be used.
+
+```bash
+$ osctl -e cluster.talos.dev memory
+NODE                TOTAL   USED   FREE   SHARED   BUFFERS   CACHE   AVAILABLE
+cluster.talos.dev   7938    1768   2390   145      53        3724    6571
+```
+
+In this case, `osctl` is interacting with `apid` running on `cluster.talos.dev` and forwarding the request to the `osd` api.
+
+If we wanted to extend our example to retrieve `memory` from another node in our cluster, we could use the command `osctl -e cluster.talos.dev -n node02 memory`.
+
+```bash
+$ osctl -e cluster.talos.dev -n node02 memory
+NODE    TOTAL   USED   FREE   SHARED   BUFFERS   CACHE   AVAILABLE
+node02  7938    1768   2390   145      53        3724    6571
+```
+
+The `apid` instance on `cluster.talos.dev` receives the request and forwards it to `apid` running on `node02` which forwards the request to the `osd` api.
+
+We can further extend out example to retrieve `memory` for all nodes in our cluster by appending additional `-n node` flags:
+
+```bash
+$ osctl -e cluster.talos.dev -n node01 -n node02 -n node03 memory
```

Comment:
Not sure what a better first impression is:

```
osctl -e cluster.talos.dev -n node01 -n node02 -n node03 memory
```
or
```
osctl -e cluster.talos.dev -n node01,node02,node03 memory 
```

Reply:
Maybe the latter since I can already hear people complaining about having to type too many characters 😄.